samples/widgets/clrpicker.cpp

```diff
@@ -119,7 +119,7 @@ wxEND_EVENT_TABLE()
 // implementation
 // ============================================================================
 
-#if defined(__WXGTK24__)
+#if defined(__WXGTK__)
```

Comment:
Not that it matters much in practice (because nobody uses GTK+ 1 any more), but this should be `__WXGTK20__`.

Reply:
Changing it to __WXGTK20__